src/containers/quiz/quiz-header/QuizHeader.tsx

```diff
@@ -22,6 +28,13 @@ type QuizHeaderProps = {
   questionsAnswered: number
   createdAt: string
   updatedAt: string
+  isNotStarted: boolean
+  quizItems: Question[]
+  settings: {
+    attemptLimit: QuizAttempt
+    timeLimit: QuizTimeLimit
+  }
+  handlePreview: (value: boolean) => void
 }
 
 const QuizHeader = ({
```

Comment:
Add React.FC generic, please
```suggestion
const QuizHeader: React.FC<QuizHeaderProps> = ({
```